src/OrleansTestingHost/TestClusterOptions.cs

```diff
@@ -19,22 +19,62 @@ namespace Orleans.TestingHost
     [Serializable]
     public class TestClusterOptions
     {
-        public enum ExtendedConfigurationOptions
+        /// <summary>
+        /// Extended options to be used as fallbacks in the case that explicit options are not provided by the user.
+        /// </summary>
+        public class FallbackOptions
         {
-            DataConnectionString,
-            TraceToConsole,
-            InitialSilosCount,
-            LogsFolder,
+            /// <summary>
+            /// Is set to true, by default the cluster will output traces in the console
+            /// </summary>
+            public bool? TraceToConsole { get; set; }
+
+            /// <summary>
+            /// Default subfolder the the logs
+            /// </summary>
+            public string LogsFolder { get; set; }
+
+            /// <summary>
+            /// Default data connection string to use in tests
+            /// </summary>
+            public string DataConnectionString { get; set; }
```

Comment:
EventHub is only in `TestDefaultConfiguration`, since the TestCluster itself does not care about it, only our tests (which is in the same vein as what @veikkoeeva was suggesting, which we should not pollute the main testing helper package with our own internal testing configuration).